### DISCUSSION TITLE
How can I fix this issue?

### DISCUSSION THREAD

**Discussion by punzzzz** [w-okada/voice-changer#1219] [General]
When i run it it usually just says for me to go to my browser and invalid url into my browser, then it fails to load the gui for the voice changer and just shows a white screen, I think it might be waiting for me to enter the url but i've also seen tutorials saying i need python or anaconda for it to work. What should I do? (I use a 2021 iMac (M1 chip))

**Comment by AnnoyingFox** [comment]
Did you run the software through the start.bat file?

**Reply by punzzzz** [reply]
wdym like i run it then i use the Start https command

**Reply by AnnoyingFox** [reply]
No, you have to start the software directly using the start.bat file (not the https one). Don't run the software opening any of the executable files.

**Reply by punzzzz** [reply]
alr i'll try that, thanks!

**Reply by punzzzz** [reply]
wait i don't have the start.bat file you are mentioning. Is this way for Mac?

**Reply by AnnoyingFox** [reply]
Oops, I think I messed my reply with someone else asking for something similar. Yeah, my reply was for Windows PC, sorry. I do not have a Mac but this has been taken from the tutorial. It's very similar to how it's done in Windows.

> Mac version
> It is launched as follows.
> 
> Unzip the downloaded file.
> 
> Next, run MMVCServerSIO by hold down the control key and clicking it (or right-click to run it). If a message appears stating that the developer cannot be verified, run it again by holding down the control key and clicking it (or right-click to run it). The terminal will open and the process will finish within a few seconds.
> 
> Next, execute the startHTTP.command by holding down the control key and clicking on it (or you can also right-click to run it). If a message appears stating that the developer cannot be verified, repeat the process by holding down the control key and clicking on it (or perform a right-click to run it). A terminal will open, and the launch process will begin.
> 
> In other words, the key is to run both MMVCServerSIO and startHTTP.command. Moreover, you need to run MMVCServerSIO first.

**Reply by punzzzz** [reply]
Alr thanks!

**Reply by punzzzz** [reply]
This is kind of just what happens when i do it directly like the instructions say. I've left it open for about 2 days and it just shuts down itself after a while. It loads just like the actual voicechanger window won't load, its just a blank screen
<img width="1337" alt="Screenshot 2024-06-03 at 6 48 05 PM" src="https://github.com/w-okada/voice-changer/assets/170936792/a032ed12-6254-4b78-9551-a908d03c204f">

**Reply by AnnoyingFox** [reply]
Maybe read the full tutorial. There may be hints of what to do if something like that happens, not sure.

**Reply by punzzzz** [reply]
alright thanks!

**Reply by punzzzz** [reply]
Yea don't think anything's working.
